Adds test back to show bug-28 (unparser deadlock) is fixed

mbeckerle opened the pull request:
dev branch, and this remains a snapshot version until daffodil 3.4.0 is released.

stevedlawrence commented:
+1 though this PR contains two commits, I think one is from another merged commit.

mbeckerle committed:
Version 1.0.5-SNAPSHOT fixes bug 28 deadlock

Requires new daffodil 3.4.0 (or snapshot of that with githash bd46fdb)

Because Daffodil 3.4.0 is still snapshot, this remains a
snapshot version on the dev branch.

https://github.com/DFDLSchemas/mil-std-2045/issues/28

mbeckerle commented:
Fixed the extra commit. Dev was behind 1 commit.
